review/tests/snapshot_tests.R

```diff
@@ -0,0 +1,111 @@
+    expect_snapshot_file("inst/markdown/11_address_matching.md")
+    expect_snapshot_file("inst/markdown/12_feedback.md")
+    expect_snapshot_file("inst/markdown/13_annex.md")
+    expect_snapshot_file("inst/markdown/final_thoughts.md")
+  })
+})
+
+# 2. Comment the above block out now
+
+# 3. Uncomment the below block, ready for future changes
+
+# 4. When you make changes to the files, use 'Run Tests' button above right
+# test_that("markdown text is as expected", {
+#   with_dir(find_package_root_file(), {
+#     source("review/scripts/word_to_md.R")
+# 
+#     expect_snapshot_file("review/temp/01_headline_figures.md")
+#     expect_snapshot_file("review/temp/02_patients_age_gender.md")
+#     expect_snapshot_file("review/temp/03_patients_imd.md")
+#     expect_snapshot_file("review/temp/04_metrics_ch_type.md")
+#     expect_snapshot_file("review/temp/05_metrics_age_gender.md")
+#     expect_snapshot_file("review/temp/06_geo_ch_flag.md")
+#     expect_snapshot_file("review/temp/07_ch_flag_drug.md")
+#     expect_snapshot_file("review/temp/08_geo_ch_flag_drug.md")
+#     expect_snapshot_file("review/temp/09_metrics_1.md")
+#     expect_snapshot_file("review/temp/09_metrics_2.md")
+#     expect_snapshot_file("review/temp/09_metrics_3.md")
+#     expect_snapshot_file("review/temp/09_metrics_4.md")
+#     expect_snapshot_file("review/temp/10_datasets.md")
+#     expect_snapshot_file("review/temp/11_address_matching.md")
+#     expect_snapshot_file("review/temp/12_feedback.md")
+#     expect_snapshot_file("review/temp/13_annex.md")
+#     expect_snapshot_file("review/temp/final_thoughts.md")
+#   })
+# })
+
+# 5. If any differences are present (there should be if you changed the
+# files...) a shiny app can be started showing the differences by running the
+# code below in the console (you can open it in browser using the 'Show in new
+# window' button in the Viewer pane)
+#     testthat::snapshot_review('snapshot_tests/', "review/tests")
+
+# 6. Review the changes in each file one by one.
+#    If ALL changes in a file are as intended, then choose the 'Accept' option.
+#    If there are some changes that were not intended, you should fix these in 
+#    the source files. You can leave the diff viewer open and revert the changes
+#    directly by a simple copy and paste of the original content. Click 'Skip' 
+#    once the reversions are complete.
+
+# 7. Close the diff viewer once only skipped files remain.
+
+# 8. If you had some changes to revert, FIRST MAKE SURE TO COMMENT OUT the line
+#        source("review/scripts/word_to_md.R")  
+#    then run this test file again with 'Run Tests'. You should expect no
+#    differences to be found now, but if there are then repeat step 6 to 8 until
+#    none are found.
+
+# 9. If you had to comment out the line
+#        source("review/scripts/word_to_md.R")
+#    due to reversions, uncomment it now so it is ready for the next set of 
+#    changes.
```

Comment:
This test file is a bit finicky, requiring manual amendment to comment/uncomment things depending on what 'test' to run. It could perhaps be made better by creating some functions that do each bit. Actually that sounds a good idea, might try it before final PR is ready to review.

Reply:
Thinking again on the best approach to using the review workflow code...

@kygoffe Would you mind, when reviewing, trying to create a simple app using the markdown as text workflow? It would be handy to check that my instructions are clear and complete. Also, feedback on how hard (or not) the process is to follow would be good to get. If it turns out to be too complicated, I will have a go at doing the code as separate functions, but don't want to spend the time to do it if it is already OK to use (spent too long already...)!